### PR TITLE
Simplify calling generate data function

### PR DESCRIPTION
**Pull request by Siegrift**
* You can call `(generate-data)` in the server repl and it will generate some data. Not very much, but at least this function is not forgotten :)

* I also removed some unused methods and imports - it seems they might be used in the future (once namebazaar is repaired), but it will be available in git anyways.

### CODE REVIEW

File: src/name_bazaar/shared/smart_contracts.cljs [hunk 1]
```diff
@@ -33,4 +33,4 @@
   :address "0x2cfd99770f16cce3463fd376eab978632a501a58"},
  :auction-offering
  {:name "AuctionOffering",
-  :address "0xb02c63815fd3a90def33a4611070b2dd417a04f8"}})
\ No newline at end of file
+  :address "0xb02c63815fd3a90def33a4611070b2dd417a04f8"}})
```
Comment (ladislavdubravsky) [inline src/name_bazaar/shared/smart_contracts.cljs]: I think we should avoid commit noise in this file.

**Comment by ladislavdubravsky** [comment]
I'm missing motivation for these actions. We have two functions on same level of seeming unusedness and decide to promote one to more attention and remove the other one.

What are you using generate-data for? Did you test it? Then maybe add the usecase to README.

Concerning deploy-to-mainnet, are you sure it's not useful for some testing? It seems like a learning resource at least.

File: src/name_bazaar/shared/smart_contracts.cljs (hunk 1 above)
Comment (Siegrift) [reply src/name_bazaar/shared/smart_contracts.cljs]: Yeah, I really dislike that we cannot ignore this file in git though...

**Comment by Siegrift** [comment]
Yeah I tested, it will generate some auction and offering, it's not very thorough, but it can be updated later if needed.
I'll add a small note to README as well, thanks.

> Concerning deploy-to-mainnet, are you sure it's not useful for some testing? It seems like a learning resource at least.
Yeah,it's definitely not used. But on the other hand, it might be called manually from `repl` during deploy. I am bit sad it doesn't have documentation about how it's used.

File: README.md [hunk 2]
```diff
@@ -51,6 +51,7 @@ folder and try to start the server again)_
 ```bash
 # In the figwheel server REPL run:
 (redeploy)
+(generate-data)
```
Comment (ladislavdubravsky) [inline README.md:55]: Aha, interesting. The section title indicates maybe this is the intended usage. I'd add an `;; and optionally` line above this, but I leave that up to you.

**Review by ladislavdubravsky — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (Siegrift) [reply README.md:55]: Yeah, good idea.

**Pull request merged**